docs/hooks/template.md

```diff
@@ -8,7 +8,7 @@
 
 ## Workflow
 
-<mark>Describe when this hook occurs in a workflow. Hook creators SHOULD include as much detail and clarity as possible to minimize any ambiguity or confusion amongst implementors.</mark>
+<mark>Describe when the `hook-name-expressed-as-noun-verb` hook occurs in a workflow. Hook creators SHOULD include as much detail and clarity as possible to minimize any ambiguity or confusion amongst implementers. The hook name should take the form `noun-verb`, such as `encounter-start`, or `order-select`.</mark>
```

Comment:
Can we link this page back to ../../specification/current/#hook-definition-format ?  I'm fine re-stating the noun-verb requirement, but the template should link to where we've documented expectations for what is included.

Reply:
done.